src/main/java/seedu/ddd/model/Model.java

```diff
@@ -59,6 +60,16 @@ public interface Model {
      */
     boolean hasContact(Contact contact);
 
+    /**
+     * Returns true if a person with the same identity as {@code person} exists in the address book.
+     */
+    boolean hasClientId(Id id);
+
+    /**
+     * Returns true if a person with the same identity as {@code person} exists in the address book.
```

Comment:
Should be `Returns true if a vendor with {@code id} exists in the address book.` instead.